Vagrantfile

```diff
@@ -0,0 +1,25 @@
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  # Every Vagrant virtual environment requires a box to build off of.
+  config.vm.box = "precise64"
```

Comment:
If you change this to

```
config.vm.box = "hashicorp/precise64"
```

then you can drop the `box_url` below and it will just download it from vagrant cloud if it can't find the box locally. I think this is the recommended way of doing things these days.